config/routes.rb

```diff
@@ -9,6 +9,8 @@
 
   get 'privacy_policy', to: 'policies#facebook'
 
+  resource :contacts, only: [:create]
```

Comment:
just curious, does this line create a new route?  if so, does this new route get used (and where?)   if not, why add this line?